code/game/objects/objs.dm

```diff
@@ -8,7 +8,7 @@
 
 	var/damtype = "brute"
 	var/force = 0
-
+	var/holospawned = 0

```

Comment:
Well if i dont make a var at obj level i will have errors since the var will not exist if its not holo